### ISSUE TITLE
La APP se cierra al tener el plugin instalado

### ISSUE THREAD

**Issue by anuskaoo** [admob-plus/admob-plus#580]
EN ANdroid 12, la APP se cierra al tener el plugin instalado. He seguido las instrucciones al pie de la letra. Incluso descargo la app de  ejemplo y me pasa lo mismo. No sé cómo solucionarlo. ALguien me puede ayudar?
Gracias

**Comment by ratson** [comment]
Did you run `npx admob-plus doctor`?

**Comment by anuskaoo** [comment]
Yes but I don`t understand the message. I removed the platform and I put it again and It work. But now the ads don't show at the first time. I don't know why. If I put a button to call the function, it works fine (although I have to press it several times). The function doesn't run on the deviceready and I don't know why. Thanks for helping

**Comment by ratson** [comment]
Are you using the testing ads? If not, the ad slot may not be filled as there maybe no available ads to be shown.
It would be helpful if you could create an example app for me to check.

**Comment by ratson** [comment]
@anuskaoo Is that your issue is resolved?

**Comment by EMI-INDO** [comment]
@ratson 
@anuskaoo 

https://github.com/admob-plus/admob-plus/discussions/587

**Comment by EYALIN** [comment]
@ratson you can close this ticket
